cli.js

```diff
@@ -55,6 +56,10 @@ const cli = meow(`
 			type: 'string',
 			alias: 't'
 		},
+		bare: {
+			type: 'boolean',
+			alias: 'n'
```

Comment:
Why `-n`? Shouldn't it be `-b`?

And I don't really think it's common enough of a use-case to warrant an alias.

Reply:
It's the same flag as `echo`.

Reply:
Since the main idea appear to support `-n` like echo, which I like, and the `--bare` seemed really only to make meow happy, I’d like to suggest changing `--bare` to `--nonewline`, which I think is more descriptive and maybe resolves the question about non-matchiness of short and long flags from @sindresorhus better.

Reply:
> --no-newline

👍

Reply:
`meow` doesn't support this, at least I can't figure out how to get the flag to be `--no-newline` and not `--noNewline`.

@sindresorhus please advise.